lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -325,6 +331,14 @@ def main():
                     image_table['image'][ii] = image
                     image_table['peak_time'][ii] = peak_time
 
+                    # use CatB pedestals to estimate the picture threshold
+                    if args.pedestal_cleaning:
+                        ped_charge_mean_pe = catB_pedestal_mean[calib_idx][selected_gain, pixel_index]
+                        ped_charge_std_pe = catB_pedestal_std[calib_idx][selected_gain, pixel_index]
```

Comment:
The conversion to p.e. is not done before

```
ped_charge_mean_pe = catB_pedestal_mean[calib_idx][selected_gain, pixel_index] * dc_to_pe
ped_charge_std_pe = catB_pedestal_std[calib_idx][selected_gain, pixel_index] * dc_to_pe
```

Reply:
the pedestals are extracted from calibrated waveforms, so this is not necessary